app/src/main/java/com/telnyx/webrtc/sdk/ui/MainActivity.kt

```diff
@@ -205,6 +217,34 @@ class MainActivity : AppCompatActivity() {
             })
     }
 
+    private fun observeWsMessage() {
```

Comment:
This feels kind of strange to me considering we are already observing a SocketResponse in the same activity.

I understand that that uses the ReceivedMessageBody class which doesn't contain all the message types but then I think we need to either add all messages there and add the messages as they come in from the live data we are already observing

OR we should adjust this getWsMessageResponse to simply be LiveData<JsonObject> and send back the raw Json and print/store that.